bin/run_brick.rb

```diff
@@ -13,17 +14,22 @@
 
 log.info "action=#{brick_type}_execution status=init"
 
+def get_brick_params(prefix)
```

Comment:
info: Doesn't depend on instance state (maybe move it to another class?) (UtilityFunction) (reek)